Robot.java

```diff
@@ -99,7 +91,7 @@ private void configureTalon(WPI_TalonSRX talon) {
 		talon.configContinuousCurrentLimit(30, Constants.timeOutMs); // Must be 5 amps or more
```

Comment:
#17

Reply:
```suggestion
		talon.configContinuousCurrentLimit(40, Constants.timeOutMs); // Must be 5 amps or more
```
Agreed, changing to 40